docs/en/02_Developer_Guides/19_GraphQL/01_getting_started/02_configuring_your_schema.md

```diff
@@ -78,14 +78,18 @@ relative to the project root.
 SilverStripe\GraphQL\Schema\Schema:
   schemas:
     default:
-      src: app/_graphql
```

Comment:
When I tried this syntax, I got this error:

```
Uncaught Exception SilverStripe\GraphQL\Schema\Exception\SchemaBuilderException: "The "src" node must be an array. Strings are not allowed." at /home/max/Projects/graphqlv4test/vendor/silverstripe/graphql/src/Schema/Schema.php line 1314 {"exception":"[object] (SilverStripe\\GraphQL\\Schema\\Exception\\SchemaBuilderException(code: 0): The \"src\" node must be an array. Strings are not allowed. at /home/max/Projects/graphqlv4test/vendor/silverstripe/graphql/src/Schema/Schema.php:1314)"} []
```

Maybe we want to support a single string, bit since the rest of the doc seems to indicate that the array syntax is better, I think it makes sense to just tell people to always use it.